main.go

```diff
@@ -38,13 +40,16 @@ func main() {
 		Version = "dev"
 	}
 
+	loggerOpts := logs.LoggerOpts{SuppressRequests: []logs.HttpRequestProperties{logs.NewAwsHealthCheckHttpRequestProperties("/notifications/version")}}
```

Comment:
It should be `notifications/api/version`

Reply:
Hi @petyos, thanks for point this out. This actually brings up a good point about this version endpoint not being standardized across all BBs. I think we should definitely try to keep it consistent, and since all of the other BBs just have the version endpoint on the base route I would suggest that we update this BB. 

In my last commit, I duplicated these APIs on the base path and deprecated the ones on the `/api` route: https://github.com/rokwire/notifications-building-block/pull/81/commits/2bdbba58df394322f0554e7a74de80bc992c0356. Let me know if you see any issues with this. Thanks!